packages/compass-aggregations/src/components/pipeline-results-workspace/pipeline-pagination-count.tsx

```diff
@@ -29,32 +32,67 @@ const countButtonStyles = css({
   },
 });
 
-export const PipelinePaginationCount: React.FunctionComponent<PipelinePaginationCountProps> =
-  ({ loading, count, onCount }) => {
-    const countDefinition = `
-      In order to have the final count of documents we need to run the
-      aggregation again. This will be the equivalent of adding a
-      $count as the last stage of the pipeline.
-    `;
+const countDocsContainerStyles = css({
+  display: 'flex',
+  alignItems: 'center',
+});
+
+// As we want to align the refresh button and loader icon,
+// we need to set the same size as the IconButton.
```

Comment:
This is referring to the IconButton LG type, whose styles I guess we don’t control?

Reply:
Yes. And this is mainly to avoid flicker when showing Refresh Button and Spinner.